app/Providers/RouteServiceProvider.php

```diff
@@ -52,7 +52,7 @@ public function map()
     protected function mapWebRoutes()
     {
         Route::middleware('web')
-             ->namespace($this->namespace)
+            //  ->namespace($this->namespace) // Disabled because we're using tuple notation for routes
```

Comment:
No need to keep comments, you have git history to review what has been deleten and why should be in commit description?

Reply:
Fair enough, will get rid of this altogether.